tooling/nargo/src/errors.rs

```diff
@@ -165,25 +166,21 @@ fn extract_locations_from_error<F: AcirField>(
         opcode_locations
             .iter()
             .flat_map(|resolved_location| {
+                let call_stack_id = match resolved_location.opcode_location {
+                    OpcodeLocation::Acir(idx) => *debug[resolved_location.acir_function_index]
+                        .location_map
+                        .get(&AcirOpcodeLocation::new(idx))
+                        .unwrap_or(&CallStackId::root()),
+                    // TODO: should we use acir_index here and merge the 2 call stacks?
```

Comment:
Are you talking about making one map and using an enum of `AcirOpcodeLocation` and `BrilligOpcodeLocation`?

Reply:
I think it was about having only one 'location tree' for all the acir functions, instead of one per acir function.
Not sure if it's worth it.

Reply:
Would be nice to consolidate our debug symbols and locations for one program, but yeah not sure how worth it it would be. Let's either add an issue or remove this todo though as this todo is unclear to me on its own.

Reply:
I created issue #7517 and removed the TODO.